README.md

```diff
@@ -10,14 +10,37 @@ authorization, transaction processing, and Reader management.
 Use Square's official Flutter plugin for Reader SDK to build in-person payment
 solutions on Android and iOS.
 
+## Migration from 3.x to 4.x
+
+Remove initialize method from `onCreate` method in your application class in Android portion
+
+```diff
+package <YOUR_PACKAGE_NAME>;
+
+- import com.squareup.sdk.reader.ReaderSdk;   
+import io.flutter.app.FlutterApplication;
+
+public class MainApplication extends FlutterApplication {
+
+ @Override
+ public void onCreate() {
+   super.onCreate();
+-   ReaderSdk.initialize(this);
```

Comment:
Why do we remove this line?

Reply:
Initialize method was moved into the plugin class and invoke just before any other SDK method is invoked https://github.com/square/reader-sdk-flutter-plugin/pull/93/files#diff-e62dc41d5752d7a035cf8fc089d566380a737c4f49b8b6e37f8228fa8e55c421R79

It has 2 reasons:
1. Less steps for plugin config
2. SDK required 2 Bluetooth permissions been granted before SDK is initialized. Both permissions are `dangerous` Wich means they should be requested in runtime after activity is started

Reply:
make sense!